web/next.config.mjs

```diff
@@ -42,6 +42,15 @@ const nextConfig = {
           },
         ],
       },
+      {
```

Comment:
wouldn't this cause issues with local development? perhaps best to set it in the deployment configuration

Reply:
I think this only goes into effect if you add the website to a list above.

Reply:
https://www.haydenbleasel.com/blog/next-hsts-preload